{"uuid": "5790f373-96ba-4483-9309-a149d751b853", "children": ["af94c015-68d4-4ee4-9d2c-fa0376aeba2f"], "befores": [{"name": "_Class__pytest_setup_method", "status": "passed", "start": 1562075750660, "stop": 1562075750660}], "afters": [{"name": "_Class__pytest_setup_method::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Test01' object has no attribute 'res'\n", "trace": "  File \"c:\\users\\44820\\pycharmprojects\\interface2\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 181, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\users\\44820\\pycharmprojects\\interface2\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 791, in _teardown_yield_fixture\n    next(it)\n  File \"c:\\users\\44820\\pycharmprojects\\interface2\\venv\\lib\\site-packages\\_pytest\\python.py\", line 776, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"c:\\users\\44820\\pycharmprojects\\interface2\\venv\\lib\\site-packages\\_pytest\\python.py\", line 688, in _call_with_optional_argument\n    func()\n  File \"C:\\Users\\44820\\PycharmProjects\\interface2\\venv\\case\\test_case_allure.py\", line 42, in teardown_method\n    if self.is_run and self.res:\n"}, "start": 1562075782374, "stop": 1562075782387}], "start": 1562075750660, "stop": 1562075782387}